pkg/k8s/informers.go

```diff
@@ -78,8 +79,11 @@ const (
 const (
 	// DefaultKubeEventResyncInterval is the default resync interval for k8s events
 	// This is set to 0 because we do not need resyncs from k8s client, and have our
-	// own Ticker to turn on periodic resyncs.
+	// own Ticker to turn on periodic resyncs for the MeshConfig resource.
```

Comment:
This default ticker is used for more than the just meshconfig?

Reply:
I found it: https://github.com/openservicemesh/osm/blob/68765c5158ed575f1785f6cee3a07d753be2bd9f/pkg/ticker/ticker.go#L55

Reply:
@jsturtevant I was trying to say that the ticker that we made for periodic resyncs was used just for the MeshConfig resource. Not that the DefaultKubeEventResyncInterval is used just for MeshConfig.
Let me know if the below change makes more sense!
```
	// DefaultKubeEventResyncInterval is the default resync interval for k8s events
	// This is set to 0 because we do not need resyncs from k8s client.
	// For the MeshConfig resource, we have our own Ticker to turn on periodic resyncs.
```

Reply:
yes, it took me a moment to understand being new to this area.  The new suggested comment makes it much clearer!